CONTRIBUTING/first-time-contributions.md

```diff
@@ -0,0 +1,83 @@
+# First time contributors
+
+We happily welcome our new contributors to this
+community. If you are contributing to CNCF
+for the first time it is okay if you
+to feel overwhelmed. We as a community
+are always there to help you with any problems
+you are facing. Open-source is about
+collaboration and we are always there
+for supporting each other.
+
+## Getting involved and contributing
+
+As a beginner or first timer, you might find
+difficulties in understanding where to start
+from. Don't worry! We got you.
+
+In the interest of getting more new people
+involved we have issues marked as
+[good-first-issues]. These are issues that have smaller
+scope, but are a great way to start with and
+getting acquainted with the project.
+
+The good-first-issues should also provide
+you details on how to get things resolved or
+how to proceed. If you find it is missing or
+incomplete please tag the person who created
+the issue and let them know.
+
+## Before your first PR
+
+Before you make you first PR, we would like
+you to go through the below attached resources
+for your understanding:
+
+- [How to submit contributions]
+- [Collaborating with pull requests]
+
+## Other ways of communication
+
+You can have additional questions or
+doubt about a certain issue, and we
+believe no doubt should be overlooked.
+You can reach us for help with any doubt
+you have, and we will be happy to have a
+one-on-one discussion.
+
+You can reach us via our [Mailing List].
+
+Join our slack [#tag-security].
+
+### You can reach out to our members
+<!-- cSpell:disable -->
+
+#### STAG CHAIRS
+
+- Emily Fox ([@TheFoxAtWork](https://github.com/TheFoxAtWork))
+- Brandon Lum ([@lumjjb](https://github.com/lumjjb))
+- Aradhana Chetal ([@achetal01](https://github.com/achetal01))
+
+#### Tech Leads
+
+- Justin Cappos ([@JustinCappos](https://github.com/JustinCappos))
+- Ash Narkar ([@ashutosh-narkar](https://github.com/ashutosh-narkar))
+- Andres Vega ([@anvega](https://github.com/anvega))
+
+#### STAG Chair Emeriti
+
+- Dan Shaw ([@dshaw](https://github.com/dshaw))
+- Sarah Allen ([@ultrasaurus](https://github.com/ultrasaurus))
+- Jeyappragash JJ ([@pragashj](https://github.com/pragashj))
+
```

Comment:
Recommend pulling this is out entirely and stating:
"You are also welcome to reach out to the [Security TAG leadership team](https://github.com/cncf/tag-security#members) individually, they can be found in the on the main [README](https://github.com/cncf/tag-security)."

This way we don't need to update this information in two places when terms end or new Tech leads are added.